Riot/AppDelegate.m

```diff
@@ -200,7 +200,12 @@ The current call view controller (if any).
      Prompt to ask the user to log in again.
      */
     UIAlertController *cryptoDataCorruptedAlert;
-    
+
+    /**
+     Prompt to warn the user about a new backup on the homeserver.
+     */
+    UIAlertController *wrongBackupVersionAlert;
```

Comment:
We could use a weak property to avoid setting `wrongBackupVersionAlert` to nil after dismiss

Reply:
True but that will not match with how it is currently done in this class. There are already several such UIAlertController. That needs a full rework.

Reply:
Ok we could use `__weak UIAlertController *wrongBackupVersionAlert;` next time.